stdlib/public/core/EmbeddedRuntime.swift

```diff
@@ -275,3 +275,9 @@ public func swift_deletedMethodError() -> Never {
 public func swift_willThrow() throws {
 }
 
+@_extern(c, "arc4random_buf")
+public func arc4random_buf(buf: UnsafeMutableRawPointer, nbytes: Int)
+
+public func swift_stdlib_random(_ buf: UnsafeMutableRawPointer, _ nbytes: Int) {
```

Comment:
Why not have this take an `UnsafeMutableRawBufferPointer`?

Reply:
Ah, I wanted to match the existing swift_stdlib_random's signature that we have in the C/C++ runtime, so that *other existing* users of swift_stdlib_random in the stdlib can be left as they are (to avoid unnecessary #ifs).